tests/framework/helpers/HtmlTest.php

```diff
@@ -137,6 +137,30 @@ public function testImg()
         $this->assertEquals('<img src="/example" alt="">', Html::img('/example'));
         $this->assertEquals('<img src="/test" alt="">', Html::img(''));
         $this->assertEquals('<img src="/example" width="10" alt="something">', Html::img('/example', ['alt' => 'something', 'width' => 10]));
+        $this->assertEquals('<img src="/base-url" alt="" srcset="/example-100w 100w,/example-500w 500w,/example-1500w 1500w">', Html::img('/base-url', [
+            'srcset' => [
+                '100w' => '/example-100w',
+                '500w' => '/example-500w',
+                '1500w' => '/example-1500w',
```

Comment:
Yes, why not? I think we should have at least the tests with 0, 1 and 3 options.